website/docs/components/side-nav/partials/code/component-api.md

```diff
@@ -14,6 +14,30 @@ This is the full-fledged component (responsive and animated).
   <C.Property @name="<:footer>" @type="named block">
     A named block where the content for the “footer” section of the SideNav is rendered. It yields the value of `isMinimized` too.
   </C.Property>
+  <C.Property @name="isResponsive" @type="boolean" @default="true">
```

Comment:
I might be mixing it up with the previous side nav, so let's just double check that the outcomes are what we want: 

1. when zoomed into 400% it reflows properly
2. scrolling up/down is possible and sticky content doesn't hide other interactive elements

The reason that I mention a zoom percent instead of pixels is the same reason the spec explainer doc has this note: 
![CleanShot 2023-08-08 at 14 33 07](https://github.com/hashicorp/design-system/assets/4587451/88c35670-3721-4884-adca-a2a647cdb6f9)

Reply:
> 1. when zoomed into 400% it reflows properly

See screenshot attached above.

> 2. scrolling up/down is possible 

It is already possible. But in this case, is there a specific height to support? (256px is frankly impossible to satisfy)

> and sticky content doesn't hide other interactive elements

With "other interactive elements" do you mean inside the sidenav (in which case it may happen for certain very small heights) or in the main content of the page (in which case at mobile viewports it's inevitable, since it works as overlay, triggered by the user)

Reply:
Inside the sidenav, yes. But I will file a separate issue to address this, because not only does it not have to be inevitable, but to leave it that way is to knowingly ship something that isn't WCAG conformant.